client/src/components/Channel.js

```diff
@@ -168,9 +168,9 @@ class Channel extends React.Component {
       ChannelActions.sendMessage(this.state.channelName, text);
   }
 
-  sendFile(filePath: string, buffer) {
+  sendFile(filePath: string, buffer, mimeType = 'application/octet-binary') {
```

Comment:
We could add the mimeType to `ipfs-post` which is what defines the data structure for each message: https://github.com/haadcode/ipfs-post/blob/master/src/FilePost.js